packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -667,8 +690,27 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval>
 	 * @param start - New start value, if defined. `undefined` signifies this endpoint should be left unchanged.
 	 * @param end - New end value, if defined. `undefined` signifies this endpoint should be left unchanged.
 	 * @returns the interval that was changed, if it existed in the collection.
+	 * @deprecated - call IntervalCollection.change with an object containing the desired parameters
 	 */
 	change(id: string, start?: number, end?: number): TInterval | undefined;
+	/**
+	 * Changes the endpoints of an existing interval.
+	 * @param id - Id of the interval to change
+	 * @param start - New start value, if defined. `undefined` signifies this endpoint should be left unchanged.
+	 * @param end - New end value, if defined. `undefined` signifies this endpoint should be left unchanged.
+	 * @returns the interval that was changed, if it existed in the collection.
```

Comment:
When you update the doc here, be sure to specify that the `props` argument is a delta over the existing props. This is probably best explained with examples, i.e. if `interval` currently has props `{ a: 5 }` and you call `interval.change(interval.getId(), { props: { b: 10 } })`, the new props will be `{ a: 5, b: 10 }`.